Best practices/React/Testing.md

```diff
@@ -98,6 +98,48 @@ Another benefit of testing React components is that they typically abstract away
   describe('<MyComponent />', () => {});
   ```
 
+* When there are multiple tests for the same prop, they should be wrapped in a single describe block.
+
+  ```js
+  // bad
+  describe('<MyComponent />', () => {
+    it('disabled gets passed to <MySubComponent />', () => {});
+    it('renders if disabled is true', () => {});
+    it('does not render if disabled is false', () => {});
+  });
+  
+  // good
+  describe('<MyComponent />', () => {
+    describe('disabled', () => {
+      it('gets passed to <MySubComponent />', () => {});
+      it('renders if true', () => {});
+      it('does not render if false', () => {});
+    });
+  });
+  ```
+
+* The statement in an it block should use present tense without repeating what is being described.
```

Comment:
I think this should go in the main testing guide (sorry if I am waffling)

Reply:
No worries.